test/expected/plan_ordered_append.out

```diff
@@ -0,0 +1,237 @@
+         ->  Index Scan using _hyper_1_2_chunk_ordered_append_time_idx on _hyper_1_2_chunk
+               Index Cond: ("time" > 'Fri Jan 07 00:00:00 2000 PST'::timestamp with time zone)
+(6 rows)
+
+-- test query with time_bucket
+:PREFIX SELECT
+  time_bucket('1d',time), device_id, value
+FROM ordered_append
+ORDER BY time ASC LIMIT 1;
+                                                QUERY PLAN                                                
+----------------------------------------------------------------------------------------------------------
+ Limit
+   ->  Result
+         ->  Append
+               ->  Index Scan Backward using _hyper_1_1_chunk_ordered_append_time_idx on _hyper_1_1_chunk
+               ->  Index Scan Backward using _hyper_1_2_chunk_ordered_append_time_idx on _hyper_1_2_chunk
+               ->  Index Scan Backward using _hyper_1_3_chunk_ordered_append_time_idx on _hyper_1_3_chunk
+(6 rows)
+
+-- test query with time_bucket (should not use ordered append)
+:PREFIX SELECT
+  time_bucket('1d',time), device_id, value
+FROM ordered_append
+ORDER BY 1 LIMIT 1;
+                                                QUERY PLAN                                                
+----------------------------------------------------------------------------------------------------------
+ Limit
+   ->  Result
+         ->  Merge Append
+               Sort Key: (time_bucket('@ 1 day'::interval, ordered_append."time"))
+               ->  Index Scan Backward using ordered_append_time_idx on ordered_append
+               ->  Index Scan Backward using _hyper_1_1_chunk_ordered_append_time_idx on _hyper_1_1_chunk
+               ->  Index Scan Backward using _hyper_1_2_chunk_ordered_append_time_idx on _hyper_1_2_chunk
+               ->  Index Scan Backward using _hyper_1_3_chunk_ordered_append_time_idx on _hyper_1_3_chunk
+(8 rows)
+
+-- test query with time_bucket (should not use ordered append)
+:PREFIX SELECT
+  time_bucket('1d',time), device_id, value
+FROM ordered_append
+ORDER BY time_bucket('1d',time) LIMIT 1;
+                                                QUERY PLAN                                                
+----------------------------------------------------------------------------------------------------------
+ Limit
+   ->  Result
+         ->  Merge Append
+               Sort Key: (time_bucket('@ 1 day'::interval, ordered_append."time"))
+               ->  Index Scan Backward using ordered_append_time_idx on ordered_append
+               ->  Index Scan Backward using _hyper_1_1_chunk_ordered_append_time_idx on _hyper_1_1_chunk
+               ->  Index Scan Backward using _hyper_1_2_chunk_ordered_append_time_idx on _hyper_1_2_chunk
+               ->  Index Scan Backward using _hyper_1_3_chunk_ordered_append_time_idx on _hyper_1_3_chunk
+(8 rows)
+
+-- test query with now() should result in ordered append with constraint aware append
+:PREFIX SELECT * FROM ordered_append WHERE time > now() + '1 month'
```

Comment:
would be nice to see an append with constraint aware append with some chunks in the relation

Reply:
done